sda-commons-web-autoconfigure/src/test/java/org/sdase/commons/spring/boot/web/monitoring/TracingFeignClientIT.java

```diff
@@ -43,18 +48,32 @@
     webEnvironment = WebEnvironment.RANDOM_PORT,
     properties = {
       "test.tracing.client.base.url=http://localhost:${wiremock.server.port}/feign",
-      "spring.zipkin.base-url=http://localhost:${wiremock.server.port}/zipkin",
-      "opa.disable=true"
+      "otel.traces.exporter=zipkin",
+      "otel.exporter.zipkin.endpoint=http://localhost:${wiremock.server.port}/zipkin/api/v2/spans",
+      "opa.disable=true",
+      "otel.metrics.exporter=none",
+      "otel.logs.exporter=none",
```

Comment:
Should this be default properties in the lib?

Reply:
Seems like we need to double check, what's really needed here and if more needs to be part of the default config.